docs/api-docs/analytics/bodl-overview.md

```diff
@@ -59,6 +60,10 @@ The following script extracts storefront data from the Stencil objects available
 
 ```handlebars title="Sample Script Code Start: Initialization Script & Page Event" lineNumbers
 <script>
+    {{inject "customerId" customer.id}}
```

Comment:
This should go down within the `BODL === 'undefined'` check, after `categoryName`.